packages/editor-ui/src/api/evaluations.ee.ts

```diff
@@ -0,0 +1,64 @@
+import type { IRestApiContext } from '@/Interface';
+import { makeRestApiRequest } from '@/utils/apiUtils';
+
+// Base interface for common properties
+export interface ITestDefinitionBase {
+	name: string;
+	workflowId?: string;
+	evaluationWorkflowId?: string;
+	description?: string;
+	annotationTagId?: string;
+}
+
+export interface ITestDefinition extends ITestDefinitionBase {
+	id: number;
+}
+
+export type CreateTestDefinitionParams = Pick<
+	ITestDefinitionBase,
+	'name' | 'workflowId' | 'description' | 'evaluationWorkflowId'
+>;
+
+export type UpdateTestDefinitionParams = Partial<ITestDefinitionBase>;
+
+// Query options type
+export interface ITestDefinitionsQueryOptions {
+	includeScopes?: boolean;
```

Comment:
I think API does not support this yet

Reply:
I got these from the CLI interface: https://github.com/n8n-io/n8n/blob/e875bc55925bd5e0bac3a8930f936bbea592dd0b/packages/cli/src/evaluation/test-definitions.types.ee.ts#L16

But will remove it for now